cspell.json

```diff
@@ -52,6 +52,12 @@
         "uninstantiated",
         "urem",
         "vecmap",
+        "direnv",
+        "nixpkgs",
+        "envrc",
+        "phated",
```

Comment:
I'm guessing there is somewhere that says `TODO(phated)` -- if so, can we create an issue and remove from cspell

Reply:
I don't actually see `phated` anywhere, so going to remove and see if spellcheck screams at me